inference_perf/main.py

```diff
@@ -114,6 +115,29 @@ def main_cli() -> None:
 
     config = read_config(args.config_file)
 
+    # Set stage rates to high values if using concurrent load type
+    if config.load.type == LoadType.CONCURRENT:
```

Comment:
Thoughts on ensuring that if rate or duration are specified when config.load.type == CONCURRENT that is a validation error?

Same with when not CONCURRENT ensuring that concurrency and num_requests is not set?


I am wondering if there is better pydantic config setup here. A subclass of LoadStage, ConcurrentLoadStage, and then ensuring that the specified LoadType has matching stage types?